app/services/package.scala

```diff
@@ -14,60 +14,30 @@
-    case e: PhraseNotFound => ProcessError(s"Referenced phrase at index ${e.index} on stanza id = ${e.id} is missing", e.id)
-    case e: LinkNotFound => ProcessError(s"Referenced link at index ${e.index} on stanza id = ${e.id} is missing", e.id)
-    case e: DuplicatePageUrl => ProcessError(s"Duplicate page url ${e.url} found on stanza id = ${e.id}", e.id)
-    case e: InconsistenQuestionError => ProcessError(s"Inconsistent QuestionStanza at id ${e.id}, number of answers and next locations dont match", e.id)
-    case e: MissingWelshText => ProcessError(s"Welsh text at index ${e.index} on stanza id = ${e.id} is empty", e.id)
-    case e: VisualStanzasAfterQuestion => ProcessError(s"Visual stanza with id = ${e.id} found following a Question stanza", e.id)
-    case e: UnknownStanza => ProcessError(s"Unsupported stanza type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: UnknownCalloutType => ProcessError(s"Unsupported CalloutStanza type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: UnknownValueType => ProcessError( s"Unsupported ValueStanza Value type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: UnknownCalcOperationType => ProcessError(s"Unsupported CalculationStanza operation type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: UnknownTestType => ProcessError( s"Unsupported ChoiceStanza test type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: UnknownInputType => ProcessError( s"Unsupported InputStanza type ${e.typeName} found at stanza id ${e.id}", e.id)
-    case e: ParseError => ProcessError(s"Unknown parse error ${e.errs.map(_.messages.mkString(",")).mkString(",")} at location ${e.jsPath.toString}", "")
-    case e: FlowParseError => ProcessError(s"Process Flow section parse error, reason: ${e.msg}, stanzaId: ${e.id}, target: ${e.arg}", e.id)
-    case e: MetaParseError => ProcessError(s"Process Meta section parse error, reason: ${e.msg}, target: ${e.id}", "")
-    case e: PhrasesParseError => ProcessError(s"Process Phrases section parse error, reason: ${e.msg}, index: ${e.id}", "")
-    case e: LinksParseError => ProcessError(s"Process Links section parse error, reason: ${e.msg}, index: ${e.id}", "")
-    case e: SharedDataInputStanza => ProcessError(s"Input/Question Stanza ${e.id} used on multiple pages ${e.pages}", e.id)
-    case e: PageRedirectNotSupported => ProcessError(s"Use of ChoiceStanza ${e.id} as a page redirect not supported", e.id)
-  }
-
-  implicit def processErrs(errs: List[GuidanceError]): List[ProcessError] = errs.map(toProcessErr)
-
-  def guidancePages(pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder, jsObject: JsObject): RequestOutcome[(Process, Seq[Page], JsObject)] =
+  def guidancePages(pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder, jsObject: JsObject)(implicit c: AppConfig): RequestOutcome[(Process, Seq[Page], JsObject)] =
```

Comment:
style warning in intellij - line exceeds 160 characters

Reply:
updated